pkg/apis/eventing/v1alpha1/source_types.go

```diff
@@ -145,6 +163,26 @@ func (ss *SourceStatus) MarkDeprovisioned(reason, messageFormat string, messageA
 	sourceCondSet.Manage(ss).MarkFalse(SourceConditionProvisioned, reason, messageFormat, messageA...)
 }
 
+// MarkProvisioned sets the condition that the source has had its backing resources created.
```

Comment:
Nit: Comment needs to be fixed for this func

Reply:
thanks!